core/src/main/java/org/elasticsearch/cluster/metadata/IndexTemplateMetaData.java

```diff
@@ -122,11 +125,11 @@ public String getName() {
         return this.name;
     }
 
-    public String template() {
+    public List<String> template() {
```

Comment:
Maybe this should become `templates`? I really wish we'd called this `patterns` instead.... Could you see how horrible that change'd make the code?